assignment2/assignment2/Presentation/FriendList/FriendListViewController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  FriendListViewController.swift
+//  assignment2
+//
+//  Created by 이승헌 on 2022/10/12.
+//
+
+import UIKit
+
+let deviceWidth: CGFloat = UIScreen.main.bounds.width
+let deviceHeight: CGFloat = UIScreen.main.bounds.height
+
+class FriendListViewController: UIViewController {
+
+    private let headView: UIView = {
+        let view: UIView = UIView()
+        
+        return view
+    }()
+    
+    private let titleLabel: UILabel = {
+        let label: UILabel = UILabel()
+        label.text = "친구"
+        label.font = .boldSystemFont(ofSize: 22)
+        
+        return label
+    }()
+    
+    private let settingButton: UIButton = {
+        let button: UIButton = UIButton()
+        button.setImage(UIImage(named: "setting"), for: .normal)
+        
+        return button
+    }()
+    
+    private let contentView: UIView = {
+        let view: UIView = UIView()
+        
+        return view
+    }()
```

Comment:
요 부분처럼 초기화 구분에서 아무것도 하지 않는다면 
```suggestion
    private let contentView = UIView()
```
이렇게 만들어도 될 것 같아요~